### DISCUSSION TITLE
[上传游戏]天穗之咲稻姬

### DISCUSSION THREAD

**Discussion by nukemiko** [Gamuxorg/bbs#292] [投递和请求游戏]
游戏名称：天穗之咲稻姬
游戏名称(英文)：Sakuna: Of Rice and Ruin
下载地址 :（如果有的话）：
- https://cloud.189.cn/t/Mz2I3uFnyeme (访问码:dar9)
- https://store.steampowered.com/app/1356670

备注：
- 这是自己打包的64位Windows游戏，内置了纯64位wine环境和dxvk。
- 下载后直接运行即可。初次运行会在`$HOME/.local/share/SakunaOfRiceAndRuin/`下面部署一个wine容器，并且解压部分游戏文件；这个路径同时也是存档的位置。
- 在命令行下加上`--help`选项查看除了启动游戏之外的用法。

**Comment by hyh19962008** [comment]
你好，可以尝试直接使用站内的投稿功能，在任意游戏页面右上角有一个“投稿”功能。
另外可以使用Steam API进行快速创建：www.linuxgame.cn/wp-json/gamux/v1/steam/appid?use_win=true
将appid替换成steam的appid，添加?use_win=true参数当游戏只有win版本时

**Comment by hyh19962008** [comment]
另外，链接无法访问，显示审核中

**Comment by nukemiko** [comment]
> 另外，链接无法访问，显示审核中

正在着手解决这个问题



> 你好，可以尝试直接使用站内的投稿功能，在任意游戏页面右上角有一个“投稿”功能。 另外可以使用Steam API进行快速创建：www.linuxgame.cn/wp-json/gamux/v1/steam/?use_win=true 将替换成steam的appid，添加?use_win=true参数当游戏只有win版本时

你们的API我无法使用，提示：抱歉，您不能编辑此项目

**Comment by hyh19962008** [comment]
需要先在站内了登陆之后才可以

**Comment by nukemiko** [comment]
> 需要先在站内了登陆之后才可以

已经登录但仍然显示“抱歉，您不能编辑此项目”这个消息

**Comment by hyh19962008** [comment]
首先在主页登录，按F12，切换到Console选项卡，敲 wpApiSettings.nonce 回车，获得一串值
www.linuxgame.cn/wp-json/gamux/v1/steam/1356670?_wpnonce=xxx&use_win=true
将xxx替换成获得的值

**Comment by nukemiko** [comment]
> 首先在主页登录，按F12，切换到Console选项卡，敲 wpApiSettings.nonce 回车，获得一串值 [www.linuxgame.cn/wp-json/gamux/v1/steam/1356670?_wpnonce=xxx&use_win=true](http://www.linuxgame.cn/wp-json/gamux/v1/steam/1356670?_wpnonce=xxx&use_win=true) 将xxx替换成获得的值

已经成功上传游戏，感谢
